bin/build-boilerplate.sh

```diff
@@ -36,8 +36,17 @@ sed -i '' -e "s/$FIND_DEF_STRING/$REPLACE_DEF_STRING/g" "$CONTENTS_DIR/definitio
 # Install dependencies
 cd $CONTENTS_DIR && npm install
 
+# Allow pushing "local changes" into the zip file
+if [ "$1" == "--debug" ]; then
+  cp -R ../src/* node_modules/zapier-platform-core/src/
+  cp -R ../node_modules/zapier-platform-legacy-scripting-runner/* node_modules/zapier-platform-legacy-scripting-runner/
+  rm -rf node_modules/zapier-platform-core/node_modules
+  rm -rf node_modules/zapier-platform-legacy-scripting-runner/node_modules
+  rm -rf node_modules/zapier-platform-schema/node_modules
+fi
+
 # Build the zip file!
-zip -R ../$FILE '*.js' '*.json'
+zip -R ../$FILE '*.js' '*.json' '*.node'
```

Comment:
`*.node` is to include the binary files necessary for package `deasync`.

Reply:
How big is the new `zip` file? No problem with it?

Reply:
Addressed in cb6852f2c0dcc.